ui/src/timeMachine/utils/queryBuilder.ts

```diff
@@ -8,7 +8,8 @@ import {
   WINDOW_PERIOD,
 } from 'src/variables/constants'
 import {AGG_WINDOW_AUTO} from 'src/timeMachine/constants/queryBuilder'
-import {BuilderTagsType} from '@influxdata/influx'
+import {BuilderTagsType} from 'src/client/'
```

Comment:
I feel like we should be getting this from `@influxdata/influx` if we can?

Reply:
or imported into and then from `src/types` ?

Reply:
@ebb-tide i think `@influxdata/influx` is the older definition. `src/client` is the one from swagger and it includes the changes i made earlier regarding `BuilderTagsType` in [16001](https://github.com/influxdata/influxdb/pull/16001/files)

Reply:
You're right that `@influxdata/influx` is the older definition but with a lot of other things that come from `src/client` we have been importing into a types file and then importing into and then exporting again from `src/types` it's kinda like saying this is a type we are using in the UI- since there are so many in src/client.

Reply:
@ebb-tide so the change is that this needs to be imported from `src/types` not `src/client`?